.github/workflows/ci-release.yml

```diff
@@ -68,8 +68,10 @@ jobs:
         include:
           - args: ""
             java: 8
+            tag_suffix: "-java-11"
```

Comment:
Shouldn’t this be “-java-8” instead of “-java-11”?

Reply:
ups